lib/formtools/plugins/plugins-helpers.js

```diff
@@ -252,6 +242,50 @@ var filtersDefault = function (filters) {
 
 };
 
+var exportDefault = function (exports) {
+
+    if (exports === undefined || exports === null) {
+        return undefined;
```

Comment:
@sandytrinh, good point. I've fixed (through a forced update). You'll need to pull down the repo to see the update.